tests/test_inference.py

```diff
@@ -1917,6 +1928,24 @@ def test_partial_bad_indexes(self):
             with pytest.raises(ValueError):
                 tsinfer.match_samples(sd, a_ts, indexes=bad_samples)
 
+    def test_time_units_default_uncalibrated(self):
```

Comment:
Need another test here to verify that passing time units to ``infer`` works also.

Reply:
OK, done. When this is merged I will close #679 and #469 but open up a new issue to discuss what I think is the correct approach.